components/bl-cropper-component/src/cropper-components.js

```diff
@@ -129,15 +133,11 @@ function Button({ className, label, onClick, visibility, disabled }) {
   );
 }
 
-function ImageUploadButton({ label, onChange, visibility }) {
-  if (!visibility) {
-    return null;
-  }
-
+function ImageUploadButton({ label, onChange, visibility, addImageButtonRef }) {
   return (
-    <label>
+    <label className={ cn({ hidden: !visibility }) }>
       <span>{ label }</span>
-      <input type="file" accept="image/*" onChange={ onChange }/>
+      <input type="file" ref={ addImageButtonRef } accept="image/*" onChange={ onChange }/>
```

Comment:
Could we get rid of this component and move everything that there to HeaderToolbar?

Im think we doesn't loose something if do like this.